source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -158,10 +162,16 @@ def create_next_kraken_request(request, responses):
     """
     one_minute = 60
     if request['clockwise']:
-        soonest_departure = min(j.departure_date_time for r in responses for j in r.journeys)
+        try:
+            soonest_departure = min(j.departure_date_time for r in responses for j in r.journeys if _has_pt(j))
```

Comment:
We don't do any next call, but we return None, and then exit the loop if request is None.